packages/atomic/src/components/atomic-query-error/atomic-query-error.tsx

```diff
@@ -13,10 +13,9 @@ const noEndpointsException = 'NoEndpointsException';
 const invalidTokenException = 'InvalidTokenException';
 
 /**
- * The QueryError component takes care of handling fatal error when doing a query on the index / Search API.
- *
+ * The `atomic-query-error` component handlies fatal errors when doing a query on the index / Search API.
  * When the error is known, it displays a relevant documentation link for debugging purposes. When the error is unknown,
- * it displays a small text area with the JSON content of the error.
+ * it displays a small text area with the JSON content of the error.The `atomic-query-error` component can be defined anywhere within your search interface.
```

Comment:
The last line is true for all components, i dont think it needs to be mentioned specifically here